CLAUDE.md

```
@@ -0,0 +1,109 @@
+## Key Patterns
+
+### Component Props with Namespace
+```typescript
+export namespace ComponentName {
+  export type Props = { title: string; };
+}
+export const ComponentName: React.FC<ComponentName.Props> = (props) => { ... };
+```
+
+### Context Pattern (for feature state)
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)